docs/root/configuration/http/http_filters/grpc_http1_bridge_filter.rst

```diff
@@ -36,6 +36,10 @@ normal gRPC requests over HTTP/2.
 
 More info: wire format in `gRPC over HTTP/2 <https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md>`_.
 
+Note that statistics are also collected by the dedicated :ref:`gRPC stats filter
```

Comment:
nit: can you put this under an `.. attention::` block?

Reply:
Done.